src/engine/source/kvdb/src/kvdb.cpp

```diff
@@ -671,13 +676,26 @@ bool KVDB::write(const std::string& key,
                  const std::string& value,
                  const std::string& columnName)
 {
+    Metrics::instance().addCounterValue("Kvdb.WritesCounter", 1UL);
     return mImpl->write(key, value, columnName);
 }
 
 std::variant<std::string, base::Error> KVDB::read(const std::string& key,
                                       const std::string& columnName)
 {
-    return mImpl->read(key, columnName);
+    Metrics::instance().addCounterValue("Kvdb.ReadsCounter", 1UL);
```

Comment:
nitpick: Could you put a brief comment above the metric to know what it measures?